### PR TITLE
Prepare for 0.8.1 release

### PR DESCRIPTION
**Pull request by amit1rrr**
Changelog & version changes for 0.8.1 release.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,17 @@ command line for details.
 
 ## 0.8
 
+### [0.8.1] - 2019-02-27
+
+- Provide better error messages
+- Allow auth scope to be array or strings
+- GitHubOAuthenticator: More efficient org_whitelist check
```
Comment (willingc) [inline CHANGELOG.md]: Thanks @amit1rrr. 

I would recommend putting keywords in backticks. For example,

```
- `GitHubOauthenticator`: More efficient `org_whitelist` check
```

would render as:

`GitHubOauthenticator`: More efficient `org_whitelist` check

The user will find this makes it easier to scan the changelog.

Comment (amit1rrr) [reply CHANGELOG.md]: Good idea. Done.

**Comment by amit1rrr** [comment]
@willingc Thanks for the review. Addressed the comment and resolved the merge conflict.

@minrk Somehow _version.py has 0.9.0 in master and editing it gives me conflict. So I removed _version.py changes. Now it's purely a Changelog edit.

Steps to release (from what I can tell)
- Create 0.8.1 branch
- Update version in _version.py
- Publish build to PyPI

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,6 +10,17 @@ command line for details.
...
 ## 0.8
 
+### [0.8.1] - 2019-02-27
+
+- Provide better error messages
+- Allow auth scope to be array or strings
+- `GitHubOAuthenticator`: More efficient `org_whitelist` check
+- Use pytest-asyncio instead of pytest-tornado since latter is unmaintained
+- CILogon: New additional_username_claims config for linked identities, fallback to the primary username claim
+- `GitLabOAuthenticator`: New `project_id_whitelist` config to whitelist users who has Developer+ access to the project
```
Comment (willingc) [inline CHANGELOG.md]: has -> have

File: CHANGELOG.md [hunk 3]
```diff
@@ -10,6 +10,17 @@ command line for details.
...
+- `GoogleOAuthenticator`: Allow email domains (hosted_domain) to be a list & the field is now mandatory
```
Comment (willingc) [inline CHANGELOG.md]: Add backticks to hosted_domain `hosted_domain`

Comment (consideRatio) [inline CHANGELOG.md]:
```suggestion
- `GoogleOAuthenticator`: Allow email domains (`hosted_domain`) to be a list & the field is now mandatory
```

File: CHANGELOG.md (hunk 2 above)
Comment (consideRatio) [inline CHANGELOG.md]:
```suggestion
- `GitLabOAuthenticator`: New `project_id_whitelist` config to whitelist users who have Developer+ access to the project
```

Comment (amit1rrr) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 3 above)
Comment (amit1rrr) [reply CHANGELOG.md]: Done.

**Comment by amit1rrr** [comment]
> Have you seen the GitHub beta feature for suggesting smaller changes within a line of code @willingc? I did the same review you did utilizing it.

Yep, a pretty neat feature!

**Comment by willingc** [comment]
Thanks @amit1rrr. Hadn't seen that @consideRatio. Lot's of UI changes happening with GitHub.

**Comment by minrk** [comment]
Thanks! Released 0.8.1

**Comment by consideRatio** [comment]
Thanks!

@amit1rrr @minrk - I inspected a lot of the z2jh configuration for authenticators, and depending on how GitLab is configuring the chart currently this is more or less relevant: https://github.com/jupyterhub/zero-to-jupyterhub-k8s/issues/1167

@amit1rrr I remember inspecting how GitLab configured z2jh before, but I failed to find my way back there in the code base. Do you know where I can find this code so we can make sure the new option is configurable in the way you do it, or if we need to do some additional work on z2jh except updating oauthenticator.

**Comment by amit1rrr** [comment]
@consideRatio here's the config code you are looking for: https://gitlab.com/gitlab-org/gitlab-ce/blob/master/app/models/clusters/applications/jupyter.rb#L65

**Pull request merged**